### PR TITLE
CSCFAIRMETA-334: Unzip secrets

### PR DESCRIPTION
**Pull request by MattiasLevlin**

### CODE REVIEW

File: unzip_secrets.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+unzip qvain_ops_sec.zip
```
Comment (jppiiroinen) [inline unzip_secrets.sh]: this is not a valid shell script. Missing the https://en.wikipedia.org/wiki/Shebang_(Unix)

**Comment by jppiiroinen** [comment]
Also a question is how would it make the deployment any easier, while the deployment steps can contain line: "./unzip-secrets.sh" or just that "unzip [packagename]".

Also this raises question if that sh is executable or not.

**Comment by jppiiroinen** [comment]
Personally I feel that this just increases complexity and does not bring value to this repository. In the case that such tools are needed, there should be a separate repository which would contain internal-only scripts and helpers.

**Comment by jppiiroinen** [comment]
lets have a chat in our issue system in the comments sections and not here.

**Comment by MattiasLevlin** [comment]
> Links to repositories for metax-ops and etsin-ops, as I failed to find a similar script

https://github.com/CSCfi/etsin-ops/blob/master/reinstall_admin_deps.sh
https://github.com/CSCfi/metax-ops/blob/centos/reinstall_admin_deps.sh

**Pull request closed without merging**